controller/eventController.js

```diff
@@ -146,13 +149,40 @@ const update = (req, res) => {
     if (!user || !ValidationUtil.isUserAdmin(user.accessType)) {
         return res.status(HttpStatus.UNAUTHORIZED).json({
             message: 'Unauhorized.'
-        })
+        });
     }
 
     Event.findOneAndUpdate({ _id: new ObjectId(eventId), companyID: new ObjectId(user.companyID) },
         req.body, { runValidators: true, new: true }).then(event => {
             if (event) {
-                return res.status(HttpStatus.OK).json(event);
+                // find event participants
+                Invite.find({ eventID: event._id, responseType: ResponseTypeEnum['Accepted'] }, { userID: 1 }).then(invites => {
+                    const userIDs = invites.map(invite => invite.userID);
+                    // get user mails
+                    User.find({ _id: { $in: userIDs } }, { email: 1 }).then(users => {
+                        const emails = users.map(u => u.email);
+                        // now get admin mails for cc - this is done separately since not all admins are part of all events and 
+                        // the idea at the moment is to include all admins in the mail as cc
+                        User.find({ companyID: user.companyID, accessType: { $in: [UserTypeEnum['Admin'], UserTypeEnum['Super-Admin']] } }, { email: 1 }).then(adminUsers => {
+                            const adminEmails = adminUsers.map(adminUser => adminUser.email);
+                            if (emailUtil.sendEmail(`${event.name} - Update`, 'Event details have been updated. Check out what\' changed.', emails, adminEmails)) {
+                                return res.status(HttpStatus.OK).json(event);
+                            } else {
+                                // what to do? update went fine, but fails at sending emails
```

Comment:
Is this comment stil relevant?

Reply:
Yes.

Reply:
So at the moment the emails are not being sent away?

Reply:
`emailUtil.sendMail` will return a `boolean` value od `true` if it managed to send the emails specified. I am not sure how to handle this case where the update succeeds, but the sending of mails fails. I mean, the update went fine, it is the notification part that fails, so that confuses me.